### PR TITLE
add a page for gut analysis toolbox

### PR DESCRIPTION
**Pull request by pr4deepr**
initializing a page for the software gut analysis toolbox. Will migrate wiki from github to this page

### CODE REVIEW

File: _pages/plugins/gut-analysis-toolbox.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+* Normalise counts to the number of ganglia.
+* Count number of neuronal subtypes, such as ChAT, nNOS etc..
+* Spatial analysis using number of neighboring cells.
+* Calcium imaging analysis: Alignment of images and extraction of normalised traces
+
+**Reference**
+
+Software citation on Zenodo:
+
+Luke Sorensen, Ayame Saito, Sabrina Poon, Myat Noe Han, Adam Humenick, Keith Mutunduwe, Christie Glennan, Narges Mahdavian, Simon JH Brookes, Rachel M McQuade, Jaime PP Foong, Sebastian K. King, Estibaliz Gómez-de-Mariscal, Robert Haase, Simona Carbone, Nicholas A. Veldhuis, Daniel P. Poole, & Pradeep Rajasekhar (2022). Gut Analysis Toolbox (1.0.0). Zenodo. https://doi.org/10.5281/zenodo.6095590
```
Comment (ctrueden) [inline _pages/plugins/gut-analysis-toolbox.md:30]: You can add the DOI as a front matter field:
```
doi: 10.5281/zenodo.6095590
```
And then any other wiki page wanting to cite the plugin can write:
```liquid
{% include citation id="plugins/gut-analysis-toolbox" %}
```
And you can put the citation in this page itself via:
```liquid
{% include citation %}
```
with no arguments.

See https://imagej.net/editing/citations#citing-by-wiki-page-url for further details.

File: _pages/plugins/gut-analysis-toolbox.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title: Gut Analysis Toolbox
+description: Gut Analysis Toolbox or GAT allows the semi-automated analysis of the cells within the enteric nervous system of the gastrointestinal tract in **2D**. GAT enables quantification of enteric neurons and their subtypes in gut wholemounts. It runs in FIJI, a popular image analysis software in microscopy and uses deep learning models to segment cells of interest. 
+categories: [Analysis,Segmentation,Neuron,Gut]
+---
+
+**Gut Analysis Toolbox** enables semi-automated analysis of the neuronal and glial distribution within the gut wall (enteric nervous system). It uses a combination of `FIJI macros`, [StarDist](https://github.com/stardist/stardist) 
+and [deepImageJ](https://deepimagej.github.io/deepimagej/) under the hood to do the heavy lifting with automated segmentation. 
+
+<p align="center">
+<img src="https://github.com/pr4deepr/GutAnalysisToolbox/blob/main/wiki_images/figures/summary_figure.png" alt="GAT_overview" width="600" >
```
Comment (ctrueden) [inline _pages/plugins/gut-analysis-toolbox.md:11]: Does the `{% include img align="center" width=600 src="..." %}` not work here? See [the documentation](https://imagej.net/editing/images#alignment). Also, you probably meant https://raw.githubusercontent.com/pr4deepr/GutAnalysisToolbox/main/wiki_images/figures/summary_figure.png for the raw image, right? Note that you can commit images to this repository in the `/media/plugins` folder, if you would prefer to host it here, but either way is fine.

File: _pages/plugins/gut-analysis-toolbox.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Gut Analysis Toolbox
+description: Gut Analysis Toolbox or GAT allows the semi-automated analysis of the cells within the enteric nervous system of the gastrointestinal tract in **2D**. GAT enables quantification of enteric neurons and their subtypes in gut wholemounts. It runs in FIJI, a popular image analysis software in microscopy and uses deep learning models to segment cells of interest.
```
Comment (ctrueden) [inline _pages/plugins/gut-analysis-toolbox.md:3]: Please spell Fiji with mixed case, not all caps. I know, it's an acronym, but "Fiji" is how the project writes it.

**Review by ctrueden — COMMENTED** [review]
Thanks for this! Just some small suggestions for improvement.

**Comment by ctrueden** [comment]
@pr4deepr Nothing showstopping to prevent merge here, so I went ahead. Feel free to make additional edits according to my suggestions, or not, as you have time and interest.

**Comment by pr4deepr** [comment]
Thanks for your feedback. 
I'll incorporate those changes soon..
Appreciate the prompt and detailed responses!

**Pull request merged**